pgcli/packages/namedqueries.py

```diff
@@ -0,0 +1,39 @@
+from os.path import expanduser
+from ..config import load_config
+
+class NamedQueries:
+
+    section_name = 'named queries'
+    filename = '~/.pgclirc'
+
+    def __init__(self):
+        self.filename = expanduser('~/.pgclirc')
```

Comment:
I would prefer to remove the hard coding of ~/.pgclirc. Instead have the constructor take in a file path as it's argument. 

```
def __init__(self, filename):
    self.filename = expanduser(filename)
```